polygeist/lib/Dialect/Polygeist/Analysis/UniformityAnalysis.cpp

```diff
@@ -102,14 +135,14 @@ void UniformityAnalysis::visitOperation(
   // If the lattice on any operand isn't yet initialized, bail out.
   if (anyOfUniformityIsUninitialized(operands)) {
     LLVM_DEBUG(llvm::dbgs().indent(2)
-               << "UA: Operand(s) uniformity not yet initialized\n");
+               << "Operand(s) uniformity not yet initialized\n");
```

Comment:
IIRC, the `UA` prefix to the output was only introduced quite recently. What's the reason to remove it again?

Reply:
Because the output is coming indented by space of 2, and the 'parent' msg has the "UA" prefix.